README.md

```diff
@@ -50,16 +50,15 @@ Wagtail works with [Python 3](https://www.python.org/downloads/), on any platfor
 
 To get started with using Wagtail, run the following in a [virtual environment](https://docs.python.org/3/tutorial/venv.html):
 
-![Installing Wagtail](.github/install-animation.gif)
-
 ```sh
-pip install wagtail
-wagtail start mysite
+uv venv
```

Comment:
There's no mention of `uv` anywhere prior to this, people who don't know what it is will have trouble running it as opposed to the plain `pip`.